source-map-uploader-plugin.js

```diff
@@ -41,31 +41,32 @@ class BugsnagSourceMapUploaderPlugin {
       const stats = compilation.getStats().toJson()
       const publicPath = this.publicPath || stats.publicPath || ''
       const outputPath = compilation.getPath(compiler.outputPath)
+      const logger = compiler.getInfrastructureLogger ? compiler.getInfrastructureLogger('BugsnagSourceMapUploaderPlugin') : console
 
       const chunkToSourceMapDescriptors = chunk => {
         // find .map files in this chunk
         const maps = chunk[webpackMajorVersion >= 5 ? 'auxiliaryFiles' : 'files'].filter(file => /.+\.map(\?.*)?$/.test(file))
 
         if (!publicPath) {
-          console.warn(`${LOG_PREFIX} ${PUBLIC_PATH_WARN}`)
+          logger.warn(`${LOG_PREFIX} ${PUBLIC_PATH_WARN}`)
```

Comment:
This needs a little more work as webpack already adds the prefix so we need to omit it when the webpack logger is used